CONTRIBUTING.md

```diff
@@ -6,25 +6,30 @@ There are multiple ways to help! We rely on users around the world to help keep
 
 Please read our [Code of Conduct](CODE_OF_CONDUCT.md) before contributing. Everyone interacting with this project (or associated projects) is expected to abide by its terms.
 
+## General note
+
+The definitions for this project are housed in a submodule. Please make sure to run `git clone --recursive git@github.com/holidays/holidays`
```

Comment:
I slightly modified the makefile target to use `git submodule update --remote --recursive`, which I found does the update I expected. Can you give it a shot yourself and see if it works @ttwo32?